### ISSUE TITLE
I2P-Bote not creating vanity destinations after upgrade from 0.3 to 0.4.3 (Trac #1815)

### ISSUE THREAD

**Issue by str4d** [i2p/i2p.i2p-bote#82]
- Click ```Identities->New Identity```
- Enter ```Public name```, keep default 256bit ECC
- Type in a single letter (also tried with multiple letters), click ```Create```
- Responds with generation page, returns back to ```New Identity``` page
- No new identity was created.

Note: creating a default generated destination works as expected.

Migrated from https://trac.i2p2.de/ticket/1815
```json
{
    "status": "closed", 
    "changetime": "2016-11-27T06:06:59", 
    "description": "- Click {{{Identities->New Identity}}}\n- Enter {{{Public name}}}, keep default 256bit ECC\n- Type in a single letter (also tried with multiple letters), click {{{Create}}}\n- Responds with generation page, returns back to {{{New Identity}}} page\n- No new identity was created.\n\nNote: creating a default generated destination works as expected.", 
    "reporter": "anonimal", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1480226819508867", 
    "component": "apps/plugins", 
    "summary": "I2P-Bote not creating vanity destinations after upgrade from 0.3 to 0.4.3", 
    "priority": "minor", 
    "keywords": "I2P-Bote", 
    "version": "0.9.26", 
    "parents": "", 
    "time": "2016-07-12T02:49:50", 
    "milestone": "0.9.28", 
    "owner": "str4d", 
    "type": "defect"
}
```

**Comment by str4d** [comment]
Trac update at 20161127T03:14:03: 
* str4d commented:

> I recreated the behaviour with the vanity character `a`. This bug was caused by the addition of XSS filtering; the error message for unsupported leading characters was being stripped. Using valid leading characters, vanity destinations worked fine for me.


* str4d changed owner from "" to "str4d"
* str4d changed status from "new" to "accepted"

**Comment by str4d** [comment]
Trac update at 20161127T06:06:59: 
* str4d commented:

> Fixed in `24a4f0352a275ad640b46d526c1cd521b069d1dc`.


* str4d changed milestone from "undecided" to "0.9.28"
* str4d changed resolution from "" to "fixed"
* str4d changed status from "accepted" to "closed"
